app/models/accounting/qb/transaction_builder.rb

```diff
@@ -18,16 +18,16 @@ def initialize(qb_division = Division.root)
       end
 
       def build_for_qb(transaction)
-        if transaction.subtype?("Check")
-          build_check_for_qb(transaction)
-        else
+        if transaction.qb_object_type == "JournalEntry"
```

Comment:
It is unlikely but possible there will be legacy disbursements that are JEs, so we decide based on qb object type, not txn type